src/ui/DynamicFacet/DynamicFacetRange.ts

```diff
@@ -57,6 +57,12 @@ export class DynamicFacetRange extends DynamicFacet implements IComponentBinding
      * @examples date
      */
     valueFormat: ComponentOptions.buildStringOption({ defaultValue: DynamicFacetRangeValueFormat.number, section: 'CommonOptions' }),
+    /**
+     * The currency symbol used if the [`valueFormat`]{@link DynamicFacetRangeValueFormat.options.valueFormat} is [`currency`]{@link DynamicFacetRangeValueFormat.currency}.
+     *
+     * **Default:** `$`
+     */
+    currencySymbol: ComponentOptions.buildStringOption({ defaultValue: '$', section: 'CommonOptions' }),
```

Comment:
Could the option be more intelligently auto-configured ? 

Could we try to detect the current locale currently loaded in the UI, and autodetermine the correct symbol to use ?
And this is not directly related to this PR, but this makes me think that we could also probably do a much better job for the `valueFormat` option. We could have a better default value depending on the field configured.

For example, we should be able to auto configure `@date` field to use `valueFormat` = `date`. 

Those kind of auto configuration are priceless and saves everybody (clients and us included) a lot of headache, so it's worth the time to think before adding any new option if we could make due without any option in the first place.

Because each option we add is a possible error/problem for our clients.

Reply:
Didn't know culture was that good!